javascript-modal/main.js

```diff
@@ -6,24 +6,21 @@ var closeModalButton = document.querySelector('.close-modal-button');
 closeModalButton.addEventListener('click', closeModal);
 
 var modal = document.querySelector('.modal');
-var overlay = document.querySelector('.overlay');
 
 var container = document.querySelector('.container');
 
 function openModal() {
   if (modalOpen === false) {
-    modal.className = 'modal visible';
-    overlay.className = 'overlay visible';
-    container.className = 'container visible';
+    modal.className = 'modal';
+    container.className = 'container';
   }
   modalOpen = true;
 }
 
 function closeModal() {
   if (modalOpen === true) {
```

Comment:
Since you have this functionality in multiple different functions, a boolean flag is not needed. You can make use of a boolean flag if you only made one event handler function, but with two separate function, you can just do the work for the two separate situations without needing to check anything. If somebody clicks the "Open Modal" button, they want to open the modal every time. If somebody clicks the "No" button, they want to close the modal every time.